src/components/IPScan/Summary/index.js

```diff
@@ -243,8 +244,9 @@ const changeTitle = (
 ) => {
   if (inputRef.current.value !== '') {
     results.xref[0].name = inputRef.current.value;
+    const input = `>${inputRef.current.value} ${results.sequence}`;
```

Comment:
I wonder if the lack of a new line `'\n'` in between the name and the sequence might cause any side effects... I check around and seems ok but worry if we are missing something

Reply:
The input field in idb when the job is created for the first time is shown the same way. I think its fine. I checked if it is elsewhere. Seems okay to me